### DISCUSSION TITLE
Training data information

### DISCUSSION THREAD

**Discussion by marisbasha** [openai/whisper#1807] [General]
Hello @jongwook, I wonder if you could provide information if Whisper large-v2 was trained with the [Ami Speech Corpus](https://groups.inf.ed.ac.uk/ami/corpus/) and [TIMIT](https://catalog.ldc.upenn.edu/LDC93s1). I am writing a paper on using whisper embeddings as a learned similarity, and I need to know this information since if it's pretrained on those datasets it would make it hard to make assumptions.

**Comment by jongwook** [comment]
Hi, we didn't specifically include those two datasets in training, but some samples from them may have been mixed into the data given the scale of the dataset. I'd still consider them "out-of-distribution".

**Reply by marisbasha** [reply]
Thanks a lot!
